pages/coq-team.html

```diff
@@ -0,0 +1,486 @@
+    <div class="name"><a href="https://www.ls2n.fr/annuaire/Gaetan%20GILBERT/">Gaëtan Gilbert</a></div>
+    <div class="nickname">SkySkimmer</div>
+    <div class="position">Inria Engineer in the Coq Consortium, attached to the Gallinette Team, Nantes, France</div>
+    <div class="components">General Maintenance, Theory, CI, Kernel, Universes, SProp, Lean Importer</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/herbelin.png" alt="Hugo Herbelin" width="170px"/>
+    </div>
+    <div class="name"><a href="http://pauillac.inria.fr/~herbelin/">Hugo Herbelin</a></div><div class="nickname">herbelin</div>
+    <div class="position">Inria Researcher in the pi.r2 Team, Paris, France.</div>
+    <div class="role">Project Coordinator (2004-2016), RM: 8.0-8.5</div>
+    <div class="components">General Maintenance, Theory, Kernel, Notations, Universes, Elaboration, Unification, Standard Library, Tactics</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/silene.jpg" alt="Guillaume Melquiond" width="150px"/></div>
+    <div class="name"><a href="https://www.lri.fr/~melquion/">Guillaume Melquiond</a></div><div class="nickname">silene</div>
+    <div class="position">Inria Researcher in the Toccata Team, Saclay, France</div>
+    <div class="role">RM: 8.9, 8.14</div>
+    <div class="components">General Maintenance, <verb>vm_compute</verb>, Reals, Coquelicot, Flocq</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/ppedrot.png" alt="Pierre-Marie Pédrot" width="220px"/><br/></div>
+    <div class="name"><a href="https://www.pédrot.fr">Pierre-Marie Pédrot</a></div>
+    <div class="nickname">ppedrot</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role">RM: 8.11</div>
+    <div class="components">General Maintenance, Theory, Performance, Kernel, Elaboration, Universes, Ltac 1/2, Forcing</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/gares.jpg" alt="Enrico Tassi" width="230px"/><br/></div>
+    <div class="name"><a href="http://www-sop.inria.fr/members/Enrico.Tassi/">Enrico Tassi</a></div>
+    <div class="nickname">gares</div>
+    <div class="position">Inria Researcher in the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="role">RM: 8.13</div>
+    <div class="components">General Maintenance, STM, ssreflect, VSCoq, Mathematical Components, Coq-Elpi, Hierarchy Builder</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/Zimmi48.jpg" alt="Théo Zimmermann" width="200px"/></div>
+    <div class="name"><a href="https://www.theozimmermann.net/en/">Théo Zimmermann</a></div><div class="nickname">Zimmi48</div>
+    <div class="position">Post-Doctoral Student in the pi.r2 Team, Paris, France</div>
```

Comment:
Is a postdoc still a student? I think the correct term is:
```suggestion
    <div class="position">Post-Doctoral Researcher in the pi.r2 Team, Paris, France</div>
```

Reply:
Indeed, both are accepted (https://www.campusfrance.org/en/post-doctorate-France) but its a bit strange as in France you're not registered at university, afaik. I put researcher instead.